ui/client/reducers/graph.js

```diff
@@ -339,15 +339,15 @@ function updateAfterNodeDelete(state, idToDelete) {
   }
 }
 
-function createUniqueNodeId(initialId, usedIds, nodeCounter) {
+function createUniqueNodeId(initialId, usedIds, isCopy) {
   return initialId && !_.includes(usedIds, initialId)
     ? initialId
-    : generateUniqueNodeId(usedIds, nodeCounter)
+    : generateUniqueNodeId(initialId, usedIds, 1, isCopy)
 }
 
-function generateUniqueNodeId(usedIds, nodeCounter) {
-  const newId = `node${nodeCounter}`;
-  return _.includes(usedIds, newId) ? generateUniqueNodeId(usedIds, nodeCounter + 1) : newId
+function generateUniqueNodeId(initialId, usedIds, nodeCounter, isCopy) {
+  const newId = isCopy ? `${initialId}(${nodeCounter} copy)` : `${initialId}${nodeCounter}`
```

Comment:
can you put a space: `nodeName (1 copy)`? also maybe will be better `nodeName (copy 1)` because if we write `1 copy` then we should rather write `1st copy` otherwise it won't be grammatically correct. `copy 1` looks more technical and then gramatics are not so important

Reply:
... also IMO `Node name 1` will looks better than `Node name1`, because we want to move into rather human names than ids in this presentation...

Reply:
Now looks like:
![image](https://user-images.githubusercontent.com/26696870/70898483-0637f300-1ff5-11ea-97fa-04735275d406.png)
![image](https://user-images.githubusercontent.com/26696870/70898530-19e35980-1ff5-11ea-8d7f-d9a9e6d65659.png)

Reply:
But copy of copy? :)